ntbs-service/Startup.cs

```diff
@@ -71,6 +72,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseCookiePolicy();
 
             app.UseMvc();
+
+            Auditer.SetupAuditFramework();
```

Comment:
Agreed that we'll want injection here, too.
I wander if `ConfigureServices` isn't a better method to place this setup in?
Also if the inject context suggestion from my previous comment doesn't pan out, perhaps we can pass the connection string through this?